pkg/image/registry/imagestream/strategy.go

```diff
@@ -23,6 +23,13 @@ import (
 	imageadmission "github.com/openshift/origin/pkg/image/admission"
 	"github.com/openshift/origin/pkg/image/api"
 	"github.com/openshift/origin/pkg/image/api/validation"
+	routecache "github.com/openshift/origin/pkg/route/generated/informers/internalversion/route/internalversion"
+)
+
+// TODO: Move this to API or helpers
+const (
+	DefaultRegistryNamespace   = "default"
+	DefaultRegistryServiceName = "docker-registry"
```

Comment:
dunno if we want this to be configurable or not, i think we have this baked into oadm registry